cmd/manager/auto_shutdown_noprocess.go

```diff
@@ -0,0 +1,60 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package manager
+
+import (
+	"regexp"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/gopsutil/process"
+)
+
+type processes map[int32]*process.FilledProcess
+
+var (
+	defaultRegexps = []*regexp.Regexp{
+		regexp.MustCompile("pause|s6-svscan|s6-supervise"),
+		regexp.MustCompile("agent|process-agent|trace-agent|security-agent|system-probe"),
+	}
```

Comment:
It might become be necessary to make this configurable in the future. Which makes me realise how the current config `auto_shutdown_method: noprocess` isn't extensible if you want to specify configs for any potential shutdown method.

Wdyt about this:

```
auto_shutdown:
  noprocess:
    enabled: true
    excludedProcesses: ["foo"]
```

Reply:
Indeed it's not configurable to allow easy configuration. I thought about adding each `method` variables as `auto_shutdown.noprocess.*` if necessary to avoid the "structured" issue

Reply:
Added the structured way but reading them as unstructured